domain/network/state/types.go

```diff
@@ -62,56 +62,74 @@ type ProviderSpace struct {
+	Name string `db:"name"`
+	// UUID is the unique ID of the availability zone.
 	UUID string `db:"uuid"`
+}
 
-	// Name is the space name.
-	Name string `db:"name"`
+// AvailabilityZoneSubnet represents a row from the availability_zone_subnet
+// table.
+type AvailabilityZoneSubnet struct {
+	// UUID is the unique ID of the availability zone.
+	AZUUID string `db:"availability_zone_uuid"`
+	// SubnetUUID is the unique ID of the Subnet.
+	SubnetUUID string `db:"subnet_uuid"`
+}
 
-	// ProviderID is the space provider id.
-	ProviderID sql.NullString `db:"provider_id"`
+// SubnetRow represents the subnet fields of a single row from the
+// v_space_subnets view.
+type SubnetRow struct {
+	// UUID is the subnet UUID.
+	UUID string `db:"subnet_uuid"`
 
-	// Subnet fields
+	// CIDR is the one of the subnet's cidr.
+	CIDR string `db:"subnet_cidr"`
 
-	// SubnetUUID is the subnet SubnetUUID.
-	SubnetUUID string `db:"subnet_uuid"`
+	// VLANTag is the subnet's vlan tag.
+	VLANTag int `db:"subnet_vlan_tag"`
 
-	// SubnetCIDR is the one of the subnet's cidr.
-	SubnetCIDR string `db:"subnet_cidr"`
+	// ProviderID is the subnet's provider id.
+	ProviderID string `db:"subnet_provider_id"`
 
-	// SubnetVLANTag is the subnet's vlan tag.
-	SubnetVLANTag int `db:"subnet_vlan_tag"`
+	// ProviderNetworkID is the subnet's provider network id.
+	ProviderNetworkID string `db:"subnet_provider_network_id"`
 
-	// SubnetProviderID is the subnet's provider id.
-	SubnetProviderID string `db:"subnet_provider_id"`
+	// ProviderSpaceUUID is the subnet's space uuid.
+	ProviderSpaceUUID sql.NullString `db:"subnet_provider_space_uuid"`
 
-	// SubnetProviderNetworkID is the subnet's provider network id.
-	SubnetProviderNetworkID string `db:"subnet_provider_network_id"`
+	// SpaceUUID is the space uuid.
+	SpaceUUID sql.NullString `db:"subnet_space_uuid"`
 
-	// SubnetProviderSpaceUUID is the subnet's space uuid.
-	SubnetProviderSpaceUUID sql.NullString `db:"subnet_provider_space_uuid"`
+	// SpaceName is the name of the space the subnet is associated with.
+	SpaceName sql.NullString `db:"subnet_space_name"`
 
-	// SubnetSpaceUUID is the space uuid.
-	SubnetSpaceUUID sql.NullString `db:"subnet_space_uuid"`
+	// AZ is the availability zones on the subnet.
+	AZ string `db:"subnet_az"`
+}
 
-	// SubnetSpaceName is the name of the space the subnet is associated with.
-	SubnetSpaceName sql.NullString `db:"subnet_space_name"`
+// SpaceSubnetRow represents a single row from the v_space_subnets view.
+type SpaceSubnetRow struct {
+	// UUID is the space UUID.
+	SpaceUUID string `db:"uuid"`
+
+	// Name is the space name.
+	SpaceName string `db:"name"`
 
-	// SubnetAZ is the availability zones on the subnet.
-	SubnetAZ string `db:"subnet_az"`
+	// ProviderID is the space provider id.
+	SpaceProviderID sql.NullString `db:"provider_id"`
+
+	SubnetRow
 }
 
-// Alias type to a slice of Space/Subnet rows.
+// Alias type to a slice of Space and Subnet rows.
```

Comment:
These are not aliases; they are new types. Comment form should start with the name in any case.